main.cpp

```diff
@@ -113,21 +113,22 @@ waiter.getOrder();
 waiter.giveBill();
 
 //  7)
-//  Noun: mechanic
-//  action 1: mechanic fixes car
-mechanic.fixCar();
+//  Noun: a Big corporation 
+//  action 1:  a Big corporation codes
+bigCorporation.code();
 
-//  action 2: mechanic bills me
-mechanic.billCurrentCustomer();
-//  action 3: mechanic gets paid
-mechanic.getPaid();
+//  action 2:  a Big corporation fails
+bigCorporation.fail();
+
+//  action 3:  a Big corporation studies
+bigCorporation.study();
```

Comment:
These actions do not use `camelCase`
The requested change asked you to revise the previous noun-action set with:
- a noun that requires `camelCase`
- 3 actions that require `camelCase`
- [ ] revise